docs/install.md

```diff
@@ -35,8 +35,35 @@ $ sudo pip install lxml --upgrade
 
 That'll spew a lot of console trash, but don't worry about it.
 
+In some installations you'll need to also upgrade `setuptools` for `html5lib`'s entertainement:
+
+~~~~
+$ sudo pip install lxml setuptools --upgrade
+~~~~
+
 From here, you can follow the commons steps outlined below.
 
+
+Travis CI steps
+---------------
+
+To use bikeshed on [Travis CI](https://travis-ci.org/)'s github integration, you'll need the following `.travis.yml` commands:
+
+~~~
+sudo: false
+language: python
+python:
+  - "2.7"
+install:
+  - pip install pygments lxml setuptools --upgrade
+  - git clone https://github.com/tabatkins/bikeshed.git
+  - pip install --editable $PWD/bikeshed
+  - bikeshed update
+script:
+  # Invoke bikeshed here, at your own leisure. E.g.:
+  - find ./ -name "*.bs" -type f | xargs -I{} -t -n1 bikeshed spec {}
```

Comment:
This seems like it's just finding the first .bs file in the folder and then passing it to bikeshed, yes? No reason to be overcomplicated, then - just calling `bikeshed` with no additional arguments will do the exact same thing.